lsp/nls/src/requests/completion.rs

```diff
@@ -287,16 +298,21 @@ fn get_identifier_path(text: &str) -> Option<Vec<String>> {
 
     let result: Vec<_> = RE_SPACE.split(text.as_ref()).map(String::from).collect();
     let path = result.iter().rev().next().cloned()?;
-    // Remove quotes from a record name
-    fn remove_quotes(name: &str) -> String {
-        let mut chars = name.chars();
-        if let (Some('"'), Some('"')) = (chars.next(), chars.last()) {
-            String::from(&name[1..name.len() - 1])
-        } else {
-            String::from(name)
-        }
-    }
-    Some(path.split('.').map(remove_quotes).collect())
+
+    let path = path
+        .split('.')
+        .map(|str| {
+            let (str, is_unicode) = remove_quotes(str);
```

Comment:
the `is_unicode` name is a bit unsettling here. The documentation of  `remove_quotes` says that the return value indicate if one of the fields was quoted, isn't it? What's the relation to unicode?

Reply:
I guess most static fields are quoted because they probably have a character that isn't allowed in a Nickel identifier (e.g unicode characters). But you're right, a it might not be a unicode string. I've made changes, thank you!